Source/WebCore/html/canvas/WebGLFramebuffer.cpp

```diff
@@ -865,7 +869,12 @@ void WebGLFramebuffer::setAttachmentInternal(GCGLenum attachment, GCGLenum texTa
 
     removeAttachmentInternal(locker, attachment);
     if (texture && texture->object()) {
-        m_attachments.set(attachment, WebGLTextureAttachment::create(texture, texTarget, level, layer));
+        RefPtr<WebGLAttachment> textureAttachment = WebGLTextureAttachment::create(texture, texTarget, level, layer);
```

Comment:
This should use `auto`; the type should be `Ref` rather than `RefPtr`. Also no need to write out `RefPtr<WebGLAttachment>` in modern code.

Reply:
Done.